PythonScript/src/HelpController.cpp

```diff
@@ -14,46 +14,50 @@ HelpController::HelpController(HWND hNotepad, HWND hScintilla)
 
 void HelpController::callHelp()
 {
-	//::HtmlHelp(m_hNotepad, getFilename(), HH_DISPLAY_TOPIC, getTopicUrl().c_str());
-	std::string helpFile = getFilename();
-	bool useWeb = false;
+	std::wstring helpFile = getFilename();
 	if (helpFile.empty())
 	{
-		helpFile = "http://npppythonscript.sourceforge.net/docs/";
-		helpFile.append(PYSCR_VERSION_STRING);
-		helpFile.append("/");
-		useWeb = true;
-	}
-
-	if (useWeb)
-	{
-		helpFile.append(getTopicUrl());
-		::ShellExecuteA(m_hNotepad, "open", helpFile.c_str(), NULL, NULL, SW_SHOWDEFAULT);
+		helpFile = L"http://npppythonscript.sourceforge.net/docs/";
+		helpFile.append(WcharMbcsConverter::char2tchar(PYSCR_VERSION_STRING).get());
+		::ShellExecute(m_hNotepad, L"open", helpFile.c_str(), NULL, NULL, SW_SHOWNORMAL);
 	}
 	else
 	{
-		::HtmlHelpA(m_hNotepad, helpFile.c_str(), HH_DISPLAY_TOPIC, reinterpret_cast<DWORD_PTR>(getTopicUrl().c_str()));
+		TCHAR *pszOut = NULL;
+		DWORD cchOUT = 0;
+		HRESULT res = ::AssocQueryString(ASSOCF_INIT_IGNOREUNKNOWN, ASSOCSTR_EXECUTABLE, L".html", NULL, pszOut, &cchOUT);
+		if (res = S_FALSE)
+			pszOut = new TCHAR[cchOUT];

```

Comment:
curly braces are missing for this section

Reply:
This happens sometimes if a pythonista is trying to code in c++ ;-)
Will be fixed with next commit.

Reply:
If we're talking about (what appears to be) the single-line-30 after the IF in line 29 above, then C/C++ doesn't require curly braces...(but I always use them)...but maybe the P.S. coding standard does before code is accepted?  Also, what's the standard for number of spaces for a "tab"...above I think I see 8 (which no one really uses!) which leads me to believe that maybe there is really an actual tab character there...and this would disappoint me if THAT is the coding standard...until Notepad++ fully implements a tab-indent-space-align feature.  :-)

Reply:
If I would have put in the braces in first place you wouldn't have been confused - it is a 
complete code block (11 lines) and afaik PS is following npp coding guidelines.
My tab width is set to 4 spaces (no replacement by 4 spaces) - it seems github uses 
a width of 8 spaces.

Reply:
I just used that to bring up wrapping single lines in braces...I like that..and a mention of what the coding standard for P.S. might be.  Unfortunately, N++'s source is a real mess tab/spaces-wise,  Tab characters always end up causing a source code mess in evolving projects with different contributors.  Sorry to get this off the main track...Claudia, remember those "curlies" in the future! :-)

Reply:
It would be nice if an extension exists which allows to set the coding guidelines per project.
I haven't found one so far but to be honest, didn't really try hard.

Reply:
- editorconfig (https://editorconfig.org/) for a basic set of config parameters by the editor (for N++ https://github.com/editorconfig/editorconfig-notepad-plus-plus)
- uncrustify (http://uncrustify.sourceforge.net/) to change complete code base